player_wizard intermittent bug

@pytest.fixture(scope='function')
    def player_wizard(browser):
        """Navigate through the player wizard."""
        wizard_main = NewCharacterCampaign(browser)
        who_are_you = wizard.WhoAreYou(browser)
        ability_scores = wizard.AbilityScoresManual(browser)

        WebDriverWait(browser, 10).until(
            EC.element_to_be_clickable(
                (By.ID, wizard_main.get_started_id)
            )
        )

        wizard_main.get_started.click()

        WebDriverWait(browser, 10).until(
            EC.element_to_be_clickable(
                (By.ID, wizard_main.player_id)
            )
        )

        wizard_main.player.click()

        WebDriverWait(browser, 10).until(
            EC.element_to_be_clickable(
                (By.ID, wizard_main.next_id)
            )
        )

        wizard_main.next_.click()

        who_are_you.character_name = 'Test Char'
        who_are_you.player_name = 'Automated Testing Bot.'

        WebDriverWait(browser, 10).until(
            EC.element_to_be_clickable(
>               (By.ID, wizard_main.next_id)
            )
        )

conftest.py:138:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <selenium.webdriver.support.wait.WebDriverWait (session="32cdc511-263c-aa41-a8ed-3889b236b539")>
method = <selenium.webdriver.support.expected_conditions.element_to_be_clickable object at 0x10f035208>, message = ''

    def until(self, method, message=''):
        """Calls the method provided with the driver as an argument until the \
            return value is not False."""
        screen = None
        stacktrace = None

        end_time = time.time() + self._timeout
        while True:
            try:
                value = method(self._driver)
                if value:
                    return value
            except self._ignored_exceptions as exc:
                screen = getattr(exc, 'screen', None)
                stacktrace = getattr(exc, 'stacktrace', None)
            time.sleep(self._poll)
            if time.time() > end_time:
                break
>       raise TimeoutException(message, screen, stacktrace)
E       selenium.common.exceptions.TimeoutException: Message:

../../../.virtualenvs/selenium/lib/python3.5/site-packages/selenium/webdriver/support/wait.py:80: TimeoutException

--- a/conftest.py
+++ b/conftest.py
@@ -102,7 +102,7 @@ def dm_wizard(browser):
     wizard_main = NewCharacterCampaign(browser)
     tell_us_a_story = TellUsAStory(browser)
 
-    WebDriverWait(browser, 10).until(
+    WebDriverWait(browser, 15).until(
         EC.element_to_be_clickable(
             (By.ID, wizard_main.get_started_id)
         )
@@ -110,7 +110,7 @@ def dm_wizard(browser):
 
     wizard_main.get_started.click()
 
-    WebDriverWait(browser, 10).until(
+    WebDriverWait(browser, 15).until(
         EC.element_to_be_clickable(
             (By.ID, wizard_main.dm_id)
         )
@@ -118,7 +118,7 @@ def dm_wizard(browser):
 
     wizard_main.dm.click()
 
-    WebDriverWait(browser, 10).until(
+    WebDriverWait(browser, 15).until(
         EC.element_to_be_clickable(
             (By.ID, wizard_main.next_id)
         )
@@ -139,7 +139,7 @@ def player_wizard(browser):
     who_are_you = wizard.WhoAreYou(browser)
     ability_scores = wizard.AbilityScoresManual(browser)
 
-    WebDriverWait(browser, 10).until(
+    WebDriverWait(browser, 15).until(
         EC.element_to_be_clickable(
             (By.ID, wizard_main.get_started_id)
         )
@@ -147,7 +147,7 @@ def player_wizard(browser):
 
     wizard_main.get_started.click()
 
-    WebDriverWait(browser, 10).until(
+    WebDriverWait(browser, 15).until(
         EC.element_to_be_clickable(
             (By.ID, wizard_main.player_id)
         )
@@ -155,7 +155,7 @@ def player_wizard(browser):
 
     wizard_main.player.click()
 
-    WebDriverWait(browser, 10).until(
+    WebDriverWait(browser, 15).until(
         EC.element_to_be_clickable(
             (By.ID, wizard_main.next_id)
         )
@@ -166,7 +166,7 @@ def player_wizard(browser):
     who_are_you.character_name = 'Test Char'
     who_are_you.player_name = 'Automated Testing Bot.'
 
-    WebDriverWait(browser, 10).until(
+    WebDriverWait(browser, 15).until(
         EC.element_to_be_clickable(
             (By.ID, wizard_main.next_id)
         )

